concepts/Common/Security Considerations/20 HTML Encoding/10 Encoding User Input.md

```diff
@@ -0,0 +1,178 @@
+Text editors, such as [TextBox](/Documentation/ApiReference/UI_Components/dxTextBox/), [Autocomplete](/Documentation/ApiReference/UI_Components/dxAutocomplete/), [HtmlEditor](/Documentation/ApiReference/UI_Components/dxHtmlEditor/), do not encode user input. We recommend that you apply third-party sanitizing tools to the user input before you submit it to the server. One of the possible ways to do it is to wrap these editors into an HTML `form` element and override the `submit` event:
```

Comment:
If our customer submits values to a server, he already wrapped the editor in a form because this is the only way.

```suggestion
Text editors, such as [TextBox](/Documentation/ApiReference/UI_Components/dxTextBox/), [Autocomplete](/Documentation/ApiReference/UI_Components/dxAutocomplete/), [HtmlEditor](/Documentation/ApiReference/UI_Components/dxHtmlEditor/), do not encode user input. We recommend that you apply third-party sanitizing tools to the user input before you submit it to the server:
```